tasks/files-to-granules/schemas/output.json

```diff
@@ -0,0 +1,42 @@
+{
+  "title": "FilesToGranulesOutput",
+  "description": "Describes the output produced by the files-to-granules task",
+  "type": "object",
+  "properties": {
+    "granules": {
+      "type": "array",
+      "description": "Array of all granules",
+      "items": {
+        "type": "object",
+        "required": ["granuleId", "files"],
+        "properties": {
+          "granuleId": {
+            "type": "string"
+          },
+          "files": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": ["name"],
+              "properties": {
+                "name": {
+                  "type": "string"
+                },
+                "filename": {
+                  "type": "string"
+                },
```

Comment:
This task needs to take into account the addition of the fileType to the Granule->Files schema